resources/es/cemu.po

```diff
@@ -2013,8 +2013,8 @@ msgid "&Render upside-down"
 msgstr "&Renderizar de arriba a abajo"
 
 #: MainWindow.cpp
-msgid "&Optimize shader SPIR-V"
-msgstr "&Optimizar sombreadores SPIR-V"
+msgid "&Async compile (Vulkan)"
+msgstr "&Computación asíncrona (Vulkan)"
```

Comment:
Probably _computación_ is not the most appropriate way to translate _compilation_, according to [how Microsoft translates that term in their products](https://www.microsoft.com/en-us/language/Search?&searchTerm=compilation&langID=736&Source=true&productid=undefined) (they use _compilación_).

Reply:
Yes, it differs from Microsoft's terminology (actually, I always check it first as I said in a previous commit) most people use "computación" instead of "compilación" when referring to async compute/compile.

Reply:
Ok, good to know!